src/defaults.js

```diff
@@ -1159,7 +1159,7 @@
                 tail = _jg.perpendicularLineTo(hxy, txy, this.width);
                 cxy = _jg.pointOnLine(hxy, txy, foldback * this.length);
 
-                var d = { hxy: hxy, tail: tail, cxy: cxy },
+                var d = [hxy, tail[0], cxy, tail[1], hxy],
```

Comment:
Here I change the way we pass the arrow drawing coordinates to the svg renderer.